README.md

```diff
@@ -3,7 +3,20 @@ This repository contains the smoke tests to check that our system is working cor
 
 This is written in ruby and we are using rubocop to ensure code standards are kept consistent throughout all our repositories.
 
+To run rubocop use this command:
+
+```
+rubocop --except Style/GlobalVars,Metrics/LineLength,Metrics/MethodLength
+```
 
 ## Setup
 
-TBC.
+### Setup for local running
+
+This is to be ran in your vagrant set up using centos-dev-env.  Instructions to start are on that github repository.
+
+ 1. Start up Digital Register by doing lr-start-all.
+
+ 2. Seed the database for titles and user by running lr-seed-data.
+
+ 3. In test_variables.rb amend SMOKE_TITLE_NUMBER, SMOKE_PARTIAL_ADDRESS, and SMOKE_POSTCODE with the title you want the smoke tests to use.
```

Comment:
It would be good to one of the two:

- `SMOKE_TITLE_NUMBER`, `SMOKE_PARTIAL_ADDRESS`, and `SMOKE_POSTCODE` all default to ones that exist in dev environment
- (safer) smoke tests define no defaults for these variables